pkg/target/target.go

```diff
@@ -161,30 +161,30 @@ func (t *targetImpl) String() string {
 	steps := []string{}
 
 	if t.Garden != "" {
-		steps = append(steps, fmt.Sprintf("garden:%q", t.Garden))
+		steps = append(steps, fmt.Sprintf("garden: %q", t.Garden))
 	}
 
 	if t.Project != "" {
-		steps = append(steps, fmt.Sprintf("project:%q", t.Project))
+		steps = append(steps, fmt.Sprintf("project: %q", t.Project))
 	}
 
 	if t.Seed != "" {
-		steps = append(steps, fmt.Sprintf("seed:%q", t.Seed))
+		steps = append(steps, fmt.Sprintf("seed: %q", t.Seed))
 	}
 
 	if t.Shoot != "" {
-		steps = append(steps, fmt.Sprintf("shoot:%q", t.Shoot))
+		steps = append(steps, fmt.Sprintf("shoot: %q", t.Shoot))
 	}
 
 	if t.ControlPlaneFlag {
 		steps = append(steps, "control plane targeted")
 	}
 
 	if len(steps) == 0 {
-		return "<empty>"
+		return "<empty>\n"
 	}
 
-	return strings.Join(steps, ", ")
+	return strings.Join(steps, "\n") + "\n"
```

Comment:
as alternative, we could set the `yaml` output as default, as we do [here](https://github.com/gardener/gardenctl-v2/blob/d889ebb88b103b3cb7695fc66682907af06e5706/pkg/cmd/config/view.go#L53-L55) and get rid of the `String()` implementation as there is currently not much difference
We already have the `yaml` output as default for the `gtv` alias https://github.com/gardener/gardenctl-v2/blob/d889ebb88b103b3cb7695fc66682907af06e5706/pkg/env/templates/rc.tmpl#L11
We could then also omit the `-o yaml` addition on the alias

Reply:
Hi Peter, thanks for the suggestion! 
I agree that setting yaml as the default output format is better. I've reworked the 'target view' command to be similar to 'config view' and I've removed the I've removed the `String()` method.